wails-frontend/src/pages/backupKeyPairs.tsx

```diff
@@ -96,7 +96,7 @@ function BackupKeyPairs() {
 
   return (
     <Layout>
-      <form ref={form} onSubmit={handleSubmit}>
+      <form ref={form} onSubmit={handleSubmit} className="max-w-xs">
```

Comment:
Can you send me a picture of before result please ? 🤔

Reply:
git checkout and test by yourself ;) it's the best way to make the product reliable!

Reply:
> @maxime-carabina check the comment linked ins the description

Thank you! It's okay for me 😉